apps/ui/src/views/Space/Editor.vue

```diff
@@ -638,26 +643,21 @@ watchEffect(() => {
               >.
             </template>
           </UiComposer>
-          <div class="s-base mb-5">
-            <UiInputString
-              :key="proposalKey || ''"
-              v-model="proposal.discussion"
-              :definition="DISCUSSION_DEFINITION"
-              :error="formErrors.discussion"
-            />
-            <UiLinkPreview
-              :key="proposalKey || ''"
-              :url="proposal.discussion"
-            />
-          </div>
+          <UiInputString
+            :key="proposalKey || ''"
+            v-model="proposal.discussion"
+            :definition="DISCUSSION_DEFINITION"
+            :error="formErrors.discussion"
+          />
+          <UiLinkPreview :key="proposalKey || ''" :url="proposal.discussion" />
           <div
             v-if="
               network &&
               strategiesWithTreasuries &&
               strategiesWithTreasuries.length > 0
             "
           >
-            <h4 class="eyebrow mb-2">Execution</h4>
+            <h4 class="eyebrow mb-2 mt-4">Execution</h4>
```

Comment:
Maybe unrelated to this PR, but spacing between executions is missing

![Screenshot 2025-03-11 at 02 10 44](https://github.com/user-attachments/assets/cc011268-1f24-4aec-a46b-f929dc66aa96)